app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -28,6 +29,13 @@ class AdvancedFilter extends React.Component {
   }
 
   componentDidMount() {
+    getAllLanguages(this.props.authenticity_token, res => {
```

Comment:
This makes the language dropdown very long, not saying we should fix this now but maybe change it similar to the enrollment ones that support text search in the future